integration-tests/src/test/scala/tech/beshu/ror/integration/suites/base/BaseAdminApiSuite.scala

```diff
@@ -26,8 +26,10 @@ import tech.beshu.ror.utils.elasticsearch.{DocumentManager, IndexManager, RorApi
 import tech.beshu.ror.utils.httpclient.RestClient
 import tech.beshu.ror.utils.misc.Resources.getResourceContent
 
+import java.time.format.DateTimeFormatter
 import scala.concurrent.duration._
 import scala.language.postfixOps
+import scala.util.Try
 
 trait BaseAdminApiSuite
```

Comment:
are we able to add tests to see if we return 400 when the request body is malformed?

Reply:
![image](https://user-images.githubusercontent.com/26696870/154808720-76dd7063-8a08-4f05-89f1-de7d23ec4167.png)